torch/csrc/autograd/functions/convolution_attr.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+#include <Python.h>
+#include "convolution.h"
+#include "torch/csrc/autograd/python_cpp_function_attr.h"
+
+
+namespace torch { namespace autograd {
+
+namespace attributes {
+
+template<class T>
+PyObject* conv_stride(THPCppFunction* self, PyObject* hook)
```

Comment:
I am not sure what are the guidelines for cpp code but these getter functions look quite similar.
Could we do a macro (or the correct cpp way to do this) like `IMPLEMENT_GET_VECTOR(conv_stride, ConvForward, stride)` or `IMPLEMENT_GET_NUMBER(conv_groups, ConvForward, groups)` that could be reused for all functions?